assets/sass/modules/_header.scss

```diff
@@ -44,18 +21,60 @@
 			width: 50%;
 		}
 	} // img
+
+	// Site Title
```

Comment:
@jomurgel Can you use the WDS standard, multiple line comment style throughout this file?

```
//-----------------------------------------
// I'm a selector comment block 
//-----------------------------------------
```